Day3/Java/HammingDistance1.java

```diff
@@ -0,0 +1,29 @@
+/* @date 4/09/2020
+ * @author Shashwat Gupta (shashwatxdevelop)
+ */
+import java.util.Scanner;
+public class HammingDistance {
+
+	public static void main(String[] args) {
+		String s1, s2; int l1, l2, i, k=0; char ch = 0;
+		Scanner sc= new Scanner (System.in);
+		s1 = sc.nextLine();  //inputing string 1
+		s2 = sc.nextLine();  // imputing string 2
+		l1 = s1.length();    // finding length of string 1
+		l2 = s2.length();    // finding length of string 2
+		
+		if(l1==l2)        // comparing both lengths of strings because hamming distance works only on same length of string
+		{
+			for (i=0 ; i<l1; i++)   // for loop for extraction of characters from both of the strings
+			{ 
+				if (s1.charAt(i)== s2.charAt(i))  // checking same character of both strings
```

Comment:
This if condition is redundant.
```suggestion
				if (s1.charAt(i)!= s2.charAt(i))  // checking same character of both strings
				   k += 1;
```